packages/desktop-client/src/components/reports/ReportOptions.ts

```diff
@@ -27,6 +27,7 @@ export const defaultReport: CustomReportEntity = {
   showUncategorized: false,
   graphType: 'BarGraph',
   conditions: [],
+  selectedCategories: [],
```

Comment:
This was added to avoid an initialization bug if the app was reloaded on the /reports/custom page.

Reply:
I don't experience this issue. Is this a bug that's introduced with this PR?

Reply:
I don't think it was. Without it, I will get 

> TypeError: Cannot read properties of undefined (reading 'map') at http://localhost:3001/src/components/reports/CategorySelector.tsx:29

when reloading /reports/custom:

![image](https://github.com/actualbudget/actual/assets/1435081/3c11de2f-e8c4-4cd7-ac54-858d10624877)

Reply:
I checked master and still experience it, so it's not something I caused.

Reply:
Yea I see it too. If you refresh twice it works. Definitely need to fix the bug, but default state as all unchecked is not desirable. 


Maybe leave it for a separate bug fix PR?

Reply:
Sure. I'll remove it from here and fix it in another PR.